kfs-core/src/main/java/edu/arizona/kfs/fp/document/web/struts/GeneralErrorCorrectionAction.java

```diff
@@ -1,5 +1,7 @@
 package edu.arizona.kfs.fp.document.web.struts;
 
+import static org.kuali.kfs.sys.KFSConstants.Booleans.TRUE;

```

Comment:
Is there a reason why not to use java's Boolean.TRUE instead of the kuali's one?